python/runtime/cudaq/algorithms/py_state.cpp

```diff
@@ -8,19 +8,21 @@
 #include <pybind11/complex.h>
 #include <pybind11/stl.h>
 
+#include "LinkedLibraryHolder.h"
+#include "common/ArgumentWrapper.h"
+#include "common/Logger.h"
+#include "cudaq/algorithms/get_state.h"
 #include "py_state.h"
 #include "utils/OpaqueArguments.h"
 #include "mlir/Bindings/Python/PybindAdaptors.h"
 #include "mlir/CAPI/IR.h"
 
-#include "common/Logger.h"
-#include "cudaq/algorithms/get_state.h"
-
 namespace cudaq {
 
 void pyAltLaunchKernel(const std::string &, MlirModule, OpaqueArguments &,
                        const std::vector<std::string> &);
-
+std::tuple<cudaq::ArgWrapper, std::size_t, std::int32_t>
```

Comment:
Should this `tuple` have a name?

Reply:
It's a good idea. Added a struct in https://github.com/NVIDIA/cuda-quantum/pull/1653/commits/c497846919dbe40b66fae307afac3483794479f6 to name this tuple.